sycl/test-e2e/lit.cfg.py

```diff
@@ -755,9 +762,16 @@ def open_check_file(file_name):
 
 # Version of the driver for a given device. Empty for non-Intel devices.
 config.intel_driver_ver = {}
-for sycl_device in config.sycl_devices:
+for full_name, sycl_device in zip(
+    config.sycl_devices, parse_devices_names(config.sycl_devices)
+):
     env = copy.copy(llvm_config.config.environment)
+
+    if "v2" in full_name:
+        env["UR_LOADER_ENABLE_LEVEL_ZERO_V2"] = "1"
```

Comment:
when do we need the `format.py` change below here?
```
                     expanded += " env UR_LOADER_USE_LEVEL_ZERO_V2=1"
```

Reply:
The change in this file is needed for sycl-ls to report the adapters correctly. The change in format.py sets the env variable for each test.

Reply:
got it